app/src/main/java/org/lfenergy/compas/cim/mapping/rest/UserInfoProperties.java

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: 2021 Alliander N.V.
+//
+// SPDX-License-Identifier: Apache-2.0
+package org.lfenergy.compas.cim.mapping.rest;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithName;
+
+@ConfigMapping(prefix = "compas.userinfo")
+public interface UserInfoProperties {
```

Comment:
Almost the same Class has been added to the SCL Data Service.
Is it viable to add this to compas core?

Reply:
The class in SCL Data Service is already different. And don't think this will be a big reuse issue. No logic.
This is the disadvantage of multiple repositories.

Reply:
Fine!